packages/ui-components/src/components/advance-date-select-dropdown/advance-date-select-dropdown.tsx

```diff
@@ -0,0 +1,219 @@
+import { Component, Element, Event, EventEmitter, Fragment, h, Host, Prop, State, Watch } from '@stencil/core';
+import { isEqual, merge } from 'lodash-es';
+import {
+	ICalendarAdvanceAbsoluteTimeConfig,
+	ICalendarAdvanceRelativeTimeConfig,
+	ITextField,
+	EActionButtonType,
+	ICalendarAdvanceSelectedTime,
+	ICalendarAdvanceTime,
+	ECalendarAdvanceTimeType,
+	SelectedRange
+} from '../../types';
+import { DEFAULT_DATE_INPUT_CONFIG, DEFAULT_DROPDOWN_POSITION_OPTIONS } from './advance-date-select-dropdown.config';
+import { IAdvanceDateSelectDropdown, IAdvanceDateSelectDropdownEvents, ITimeChange } from './advance-date-select-dropdown.types';
+import {
+	formatAbsoluteSelectedTime,
+	getRelativeTimeLabel,
+	getTimeRange,
+	isAbsoluteTimeSelected,
+	isRelativeTimeSelected,
+	isTimeSelected
+} from './advance-date-select-dropdown.helper';
+import { formatTimezoneName, fromDatesRangeKey, getDefaultTimezone, getTimezonesNames } from '../../utils/date.helper';
+import { ComputePositionConfig } from '@floating-ui/dom';
+
+@Component({
+	tag: 'kv-advance-date-select-dropdown',
+	styleUrl: 'advance-date-select-dropdown.scss',
+	shadow: true
+})
+export class KvAdvanceDateSelectDropdown implements IAdvanceDateSelectDropdown, IAdvanceDateSelectDropdownEvents {
```

Comment:
I think is better to call `KvAdvancedDateSelectDropdown`. Basically change `advance` by `advanced`